src/storage/test/UpdateEdgeTest.cpp

```diff
@@ -352,6 +352,72 @@ TEST(UpdateEdgeTest, Insertable_Test) {
     EXPECT_STREQ("", boost::get<std::string>(v3).c_str());
 }
 
+
+TEST(UpdateEdgeTest, CorruptDataTest) {
+    fs::TempDir rootPath("/tmp/UpdateEdgeTest.XXXXXX");
+    std::unique_ptr<kvstore::KVStore> kv = TestUtils::initKV(rootPath.path());
+    LOG(INFO) << "Prepare meta...";
+    auto schemaMan = TestUtils::mockSchemaMan();
+    auto indexMan = TestUtils::mockIndexMan();
+    LOG(INFO) << "Write an edge with empty value!";
+
+    // partId, srcId, edgeType, rank, dstId, version
+    auto key = NebulaKeyUtils::edgeKey(0, 10, 101, 0, 11, 0);
+    std::vector<kvstore::KV> data;
+    data.emplace_back(std::make_pair(key, ""));
+    folly::Baton<> baton;
+    kv->asyncMultiPut(0, 0, std::move(data),
+        [&](kvstore::ResultCode code) {
+            CHECK_EQ(code, kvstore::ResultCode::SUCCEEDED);
+            baton.post();
+        });
+    baton.wait();
+
+    LOG(INFO) << "Build UpdateEdgeRequest...";
+    GraphSpaceID spaceId = 0;
+    PartitionID partId = 0;
+    VertexID srcId = 10;
+    VertexID dstId = 11;
+    // src = 1, edge_type = 101, ranking = 0, dst = 10001
```

Comment:
update  `dst = 11`   or remove the comment ？

Reply:
Thanks for reminding.  Let's me take care of it in next pr.